configs/hoodi/testnet.yaml

```diff
@@ -0,0 +1,1916 @@
+    - &lidoApm "0x2F78689e4534b1461F4c35C2fC65b53205f6e50f"
+    - &lidoTemplate "0xEC39F7ea18BC1c1bf3C0617fb6110dfC62d1e888"
+    - &miniMeTokenFactory "0x4d0E05677c1005C4C808cb5e824abE99b2b39497"
+    - &pinnedBeaconProxy "0xF19E48C5Ad2F8aBB452F5A6AB9cb104E70495e08"
+
+roles:
+  - &MANAGE_CONSENSUS_CONTRACT_ROLE "0x04a0afbbd09d5ad397fc858789da4f8edd59f5ca5098d70faa490babee945c3b"
+  - &MANAGE_CONSENSUS_VERSION_ROLE "0xc31b1e4b732c5173dc51d519dfa432bad95550ecc4b0f9a61c2a558a2a8e4341"
+  - &SUBMIT_DATA_ROLE "0x65fa0c17458517c727737e4153dd477fa3e328cf706640b0f68b1a285c5990da"
+  - &STAKING_CONTROL_ROLE "0xa42eee1333c0758ba72be38e728b6dadb32ea767de5b4ddbaea1dae85b1b051f"
+  - &RESUME_ROLE "0x2fc10cc8ae19568712f7a176fb4978616a610650813c9d05326c34abb62749c7"
+  - &PAUSE_ROLE "0x139c2898040ef16910dc9f44dc697df79363da767d8bc92f2e310312b816e46d"
+
+misc:
+  - &ZERO_ADDRESS "0x0000000000000000000000000000000000000000"
+  - &ZERO_BYTES32 "0x0000000000000000000000000000000000000000000000000000000000000000"
+  - &REGISTRY_BASE "0x13f43DCe264695D4118Cbc8Adb8691ABaEa0562d"
+  - &DEPOSIT_CONTRACT "0x00000000219ab540356cBB839Cbe05303d7705Fa"
+  - &MAX_UINT256 "115792089237316195423570985008687907853269984665640564039457584007913129639935"
+  - &MAX_UINT64 "18446744073709551615"
+  - &ensSubdomainRegistrarImpl "0x5ED5661F744E11A2076865810B0d7553b74468D2"
+  - &aragonIdResolver "0x3eBCA460bA62724CaE0F801210e76dBB4f3C2A1f"
+
+eoa:
+  - &depositSecurityModule "0xfF772cd178D04F0B4b1EFB730c5F2B9683B31611"
+  - &deployer "0x26EDb7f0f223A25EE390aCCccb577F3a31edDfC5"
+
+l1:
+  rpcUrl: HOODI_REMOTE_RPC_URL # https://ethereum-hoodi-rpc.publicnode.com
+  explorerHostname: api-hoodi.etherscan.io
+  explorerTokenEnv: ETHERSCAN_TOKEN
+  contracts:
+    lidoLocator:
+      name: LidoLocator
+      address: *lidoLocator
+      proxyName: OssifiableProxy
+      implementation: *lidoLocatorImpl
+      proxyChecks:
+        proxy__getAdmin: *aragonAgent
+        proxy__getImplementation: *lidoLocatorImpl
+        proxy__getIsOssified: false
+      checks:
+        accounting: *accounting
+        accountingOracle: *accountingOracle
+        burner: *burner
+        coreComponents:
+          - *executionLayerRewardsVault
+          - *oracleReportSanityChecker
+          - *stakingRouter
+          - *aragonAgent
+          - *withdrawalQueueERC721
+          - *withdrawalVault
+        depositSecurityModule: *depositSecurityModule
+        elRewardsVault: *executionLayerRewardsVault
+        lido: *lido
+        operatorGrid: *operatorGrid
+        oracleDaemonConfig: *oracleDaemonConfig
+        oracleReportComponents:
+          - *accountingOracle
+          - *oracleReportSanityChecker
+          - *burner
+          - *withdrawalQueueERC721
+          - *ZERO_ADDRESS
+          - *stakingRouter
+          - *vaultHub
+        oracleReportSanityChecker: *oracleReportSanityChecker
+        postTokenRebaseReceiver: *ZERO_ADDRESS
```

Comment:
interesting, probably we should update scratch deployment :thinking: cc: @tamtamchik

Reply:
@TheDZhon wdym? We removed the Legacy Oracle, so now it is emply.

Reply:
For now, leave it as is, and for the mainnet, we will update to a real receiver.